metaworld/envs/mujoco/env_dict.py

```diff
@@ -21,6 +21,7 @@
 from metaworld.envs.mujoco.sawyer_xyz.sawyer_drawer_close import SawyerDrawerCloseEnv
 from metaworld.envs.mujoco.sawyer_xyz.sawyer_box_close import SawyerBoxCloseEnv
 from metaworld.envs.mujoco.sawyer_xyz.sawyer_peg_insertion_side import SawyerPegInsertionSideEnv
+from metaworld.envs.mujoco.sawyer_xyz.sawyer_peg_insertion_side_v2 import SawyerPegInsertionSideEnvV2
```

Comment:
it seems some unrelated commits have creeped in?

Reply:
Yep that’s why I kept it a draft. It’s just to make rebasing easier when #102 gets merged.